CRM/Contact/BAO/Contact.php

```diff
@@ -1063,7 +1063,7 @@ public static function deleteContact($id, $restore = FALSE, $skipUndelete = FALS
     }
 
     //delete the contact id from recently view
-    CRM_Utils_Recent::delContact($id);
```

Comment:
Yep - tested all good - note that when soft deleting it gets a strike through & is removed on hard-delete